Payments/UserInterface/01a_ListAuthorizations.md

```diff
@@ -0,0 +1,299 @@
+
+**Authorization transaction "Authorization ID"**
+
+- ![Back](../../Assets/Icons/Back02.png "[Back]") (Back)   
+    Click this button to close the *Authorization "Authorization ID"* view and return to the list of authorizations. 
+
+
+
+## Authorizations &ndash; Attributes
+
+*Payments > Authorizations > Tab LIST > Select an authorization transaction > Tab Attributes*   
+
+
+![Attributes](../../Assets/Screenshots/Payments/Authorizations/AuthorizationAttributes.png "[Attributes]")
+
+In the left margin column, all available attribute groups are displayed. Click a group to display the attributes that are assigned to this group on the right side of the *Attributes* tab. If the authorization transaction contains attributes that are not assigned, the *Unassigned group* attribute group is automatically displayed in the left margin column.
+
+- ![Folders](../../Assets/Icons/Folders01.png "[Folders]") (Folders)  
+    Attribute group that contains attribute sub-groups. Click the attribute group or the arrow left to the attribute group to expand the group and to display the attribute sub-groups. The following sub-groups are available:
+    - *[Payment service provider] Incoming Payment Base Data*   
+      This folder contains attributes that are specific for the payment service provider who has sent this authorization ticket.
+    - *Incoming Payment Base Data*   
+      This folder contains general attributes that Actindo needs to query payments data. These data support standard functions of Actindo.  
+    - *Unassigned Group*   
+      This folder contains attributes that are not assigned the other groups.
+
+- ![Folder](../../Assets/Icons/Folder01.png "[Folder]") (Folder)  
+    Attribute group. Click the attribute group to display all attributes that are assigned to the selected attribute group on the right side of the *Attributes* tab.
+
+The right side of the *Attributes* tab displays all attributes that are assigned to the selected group in the left margin column. Depending on the selected attribute group, the displayed fields may vary. The attributes are imported by the payment service provider driver. All fields are read-only.   
+
+- ![Collapse/Expand](../../Assets/Icons/CollapseExpand01.png "[Collapse/Expand]") (Collapse/Expand)    
+    Click this button to collapse or expand the left margin column with the attribute groups. When the left margin is expanded and you click this button, the column is collapsed. When the column is collapsed and you click this button, the column is expanded. This is available for the *Unassigned Group* folder only.
+
+
+## Authorizations &ndash; Dependencies  
+
+*Payments > Authorizations > Tab LIST > Select an authorization ticket > Tab Dependencies*
+
+![Dependencies](../../Assets/Screenshots/Payments/Authorizations/AuthorizationDependencies.png "[Dependencies]")   
+
+
+Since a authorization transaction has no dependencies to other entities in Actindo by default, this sub-tab is not relevant.
```

Comment:
```suggestion
Since an authorization transaction has no dependencies to other entities in Actindo by default, this sub-tab is not relevant.
```
Im Wissenstransfer Video (20230510) erklärt er, dass es geben könnte... (z.B. über DataHub Exporter). Trotzdem sinnvoll die Standard Beschreibung hinzuzufügen?

Reply:
Ich denke schon